packs/chef/actions/chef_install.py

```diff
@@ -20,9 +19,10 @@ class ChefInstaller(object):
         ('-d', '--download_path', {})
     ]
 
-    def install(self, install_method, options):
+    @staticmethod
+    def install(install_method, options):
         exit_code = 0
-        installer = getattr(ChefInstaller, install_method.capitalize())(options)
+        installer = getattr(sys.modules['lib'], install_method.capitalize())(options)
```

Comment:
And I imagine that just changing the code to:

```python
installer = getattr(self, install_method.capitalize())(options)
```
and commenting out the import inside the class would work as well.